src/ai/backend/manager/api/schema.graphql

```diff
@@ -583,7 +583,10 @@ type ComputeSession implements Item {
   status_changed: DateTime
   status_info: String
   status_data: JSONString
-  status_history: JSONString
+  status_history: JSONString @deprecated(reason: "Deprecated since 24.09.0; use `status_history_log`")
+
+  """Added in 24.09.0"""
+  status_history_log: JSONString
```

Comment:
JSONstring is right?

Reply:
Yes, it is still of the `JSONString` type. For example, `status_history_log` is queried as shown below.

```
{
  "data": {
    "compute_session": {
      "status_history_log": "[{\"status\": \"PENDING\", \"timestamp\": \"2024-12-05T02:34:32.647820+00:00\"}, {\"status\": \"SCHEDULED\", \"timestamp\": \"2024-12-05T02:34:33.287036+00:00\"}]"
    }
  }
}
```